install/install.sh

```diff
@@ -41,7 +41,12 @@ NECESSARY_SPACE_GB=4
 
 # Check for docker and install it if not found
 echo "Checking for docker."
-docker --version || curl -fsSL https://get.docker.com | sh && systemctl enable docker
+docker --version || curl -fsSL https://get.docker.com | sh || (
+    echo "Failed to start docker, something is wrong."
+    echo "Please report this problem."
+    exit 1
+)
+systemctl enable docker
```

Comment:
I assume you were not able to reproduce #106, right?

Reply:
Nops, I got a network problem where my avahi work messed up with devconf and the result was that get.docker was failing. 

The install script is working as intended